server/index.js

```diff
@@ -20,5 +21,81 @@ app.get('/api/alsoBought/:id', (req, res) => {
   });
 });
 
+// Get all stock records
+app.get('/api/alsoBought/', (req, res) => {
+  Stock.readAll((err, stocks) => {
+    if (err) {
+      res.sendStatus(400);
+    } else {
+      res.status(200).json(stocks);
+    }
+  });
+});
+
+//Show the stock record in the db for the specific id
+app.get('/api/alsoBought/:id/show', (req, res) => {
+  Stock.readOne(req.params.id, (err, stock) => {
```

Comment:
might have a syntax error here, I thought "readOne" expected an object as the first parameter